bin/metrics/metrics.py

```diff
@@ -25,17 +25,17 @@ def yield_bp_q30(mqc):
     return k, v
 
 
-def pct_reads_aligned_in_pairs(mqc):
+def pct_reads_mapped(mqc):
     """
-    The percentage of reads that have been aligned as pairs.
+    The percentage of primary reads, paired or single, that are mappable to the REF sequence with MAPQ > 0 after alignment. 
```

Comment:
Metric definition should align with GA4GH documentation

Reply:
agree.
I think we can leave this ultimate cleanup/update of the docstring when PR devel - > main

Reply:
Created a dedicated issue #89